docs/elements.md

```diff
@@ -545,7 +547,7 @@ The `base` argument can also accept a special value of 0. In this case, the valu
 
 #### Integer range
 
-The valid range of values accepted by pl-integer-input is between -9007199254740991 and +9007199254740991 (between -(2^53 - 1) and +(2^53 - 1)). If you need a larger input, one option you can consider is a [`pl-string-input`](#pl-string-input-element) with a [custom grade function](question.md#question-serverpy).
+pl-integer-input can accept integers of unbounded size, however the correct answer will only be stored as the Python `int` if it is between -9007199254740991 and +9007199254740991 (between -(2^53 - 1) and +(2^53 - 1)). Otherwise, the correct answer will be stored as a string.
```

Comment:
Probably worth mentioning where this is important: in the server.py scripts for parse and grade, as well as in downloaded assessment results.

Reply:
I also forgot to mention that answers are allowed to contain underscores 🤦‍♂️

Reply:
Discussion added here: https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/736944fa7557c6023bfd70b202cbe332ea6a0c3b